### ISSUE TITLE
Не доступна веб-версия

### ISSUE THREAD

**Issue by konofeev** [Bashka/manual#2]
Доброго времени суток.
Web-версия мануала по адресу: http://188.242.76.233/ не доступна

**Comment by Bashka** [comment]
Да, к сожалению сервер канул в лету. Вполне возможно изучать исходники, а я постараюсь переформатировать их в соответствии с требованиями гитхаба чуть позже.
